mdal/frmts/mdal_selafin.cpp

```diff
@@ -159,10 +158,14 @@ void MDAL::SelafinFile::parseFile()
 
   /* 1 record containing table X (real array of dimension NPOIN containing the
      abscisse of the points)
+     AND it is here we can now if float is simple or double precision:
+     size of record / number of vertices gives the size of the float -> 4 : simple precision -> 8 : double precision
```

Comment:
Can you please rewrite the sentence, it is not (gramatically) clear what is meant